app/index.html

```diff
@@ -15,6 +15,15 @@
     <link href='https://fonts.googleapis.com/css?family=Open+Sans:400,600,300,700' rel='stylesheet' type='text/css'>
 
     {{content-for "head-footer"}}
+    <script type="text/javascript"
```

Comment:
The below links may not apply to this instance of manually editing index.html, because we will always want MathJax from a CDN (to support local development) instead of only using the library when in production. But noted for posterity: 
https://github.com/abought/ember-preprints/blob/31b35b85b6c84e15e1a9227e6e743079330c04e7/ember-cli-build.js#L36-L36

https://www.npmjs.com/package/ember-cli-inline-content